spiffworkflow-backend/src/spiffworkflow_backend/__init__.py

```diff
@@ -25,12 +25,11 @@
 from spiffworkflow_backend.services.monitoring_service import configure_sentry
 from spiffworkflow_backend.services.monitoring_service import setup_prometheus_metrics
 
-# This is necessary if you want to use use the pymysql library with sqlalchemy rather than mysqlclient.
-# This is only potentially needed if you want to run non-docker local dev.
-# See the repo's top-level README and the linked troubleshooting guide for details.
-# import pymysql;
-# pymysql.install_as_MySQLdb()
-
+# This is necessary to make sure that the pymysql library is used as the MySQLdb library
+# This is only needed if you want to run non-docker local and are using Python 3. 
+# See the repo's top-level README for details.
+import pymysql;
```

Comment:
can we leave this file unchanged? it's probably better to avoid importing and using the pure python mysql client by default. ideally it would be better to not include the pymysql dependency in pyproject.toml as well. we can debug the mysqlclient issues you were having sometime if you want.

Reply:
Oh right oops. The mysqlclient issues are all about how they get installed and built; the deps don't resolve (this has nothing to do with the app code, the app won't install locally with the package.json as-is).